packages/web-runtime/src/components/VersionCheck.vue

```diff
@@ -4,7 +4,11 @@
     <oc-spinner class="ml-1" size="xsmall" />
   </div>
   <div v-else>
-    <div v-if="!updateAvailable" class="version-check-no-updates flex items-center">
+    <div v-if="hasError" class="version-check-error flex items-center">
```

Comment:
Why is this wrapped in another div? Can't this be a `<div v-else-if="hasError" ...` one level higher?